### PR TITLE
Filter incomplete applications

### PR DESCRIPTION
**Pull request by tomknig**

### CODE REVIEW

**Review by kurdybacha — COMMENTED** [review]
Thanks for looking at this.

File: actions/applications/index.js [hunk 1]
```diff
@@ -27,6 +27,31 @@ export async function upsertApplications(records) {
...
+    {
+      $match: {
+        emailAddress: { $exists: true },
+        projectName: { $exists: true },
+        projectTweet: { $exists: true },
+        productPitch: { $exists: true },
+        founderBackground: { $exists: true },
+        evidenceOfExceptionalAbility: { $exists: true },
+        discordId: { $exists: true },
+        submittedAt: { $exists: true },
```
Comment (kurdybacha) [inline actions/applications/index.js:39]: we do not have these filed in db so please remove.

**Review by kurdybacha — CHANGES_REQUESTED** [review]

File: actions/applications/index.js (hunk 1 above)
Comment (tomknig) [reply actions/applications/index.js]: I think we have them: https://github.com/hyper-scale/race/blob/main/models/Application.js#L47-L60 or am I missing something?

Comment (tomknig) [reply actions/applications/index.js]: Ok got it. I think you are talking about the actual data we have in the db while I was talking about the schema. You are right. I'll remove these for now.

Comment (tomknig) [reply actions/applications/index.js]: Ok got it. I think you are talking about the actual data we have in the db while I was talking about the schema. You are right. I'll remove these for now.

Comment (kurdybacha) [reply actions/applications/index.js]: Yes, I was referring to the data in the DB. I am sorry for confusion. Could you also check why we get different first 5 applications on main page and leaderboard table?

**Comment by kurdybacha** [comment]
I will merge that and any follow up can be in a separate PR

**Pull request merged**